core/server/worker/src/main/java/alluxio/worker/netty/UfsFileWriteHandler.java

```diff
@@ -88,6 +88,7 @@ protected PacketWriter createPacketWriter(UfsFileWriteRequestContext context, Ch
    */
   public class UfsFilePacketWriter extends PacketWriter {
     private final UfsManager mUfsManager;
+    private CloseableResource<UnderFileSystem> mUfsClientResource;
```

Comment:
seems like we have multiple instances of `UfsFilePacketWriter` for the same request. moving this to context